### PR TITLE
Add contribute, remove user_retirement and ORA2 from tests

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-XXXX](https://openedx.atlassian.net/browse/DOC-XXXX)

I added /contribute and removed /user_retirement and /ORA2 from tests

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @jmbowman 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Squash commits

### CODE REVIEW

File: run_tests.sh [hunk 1]
```diff
@@ -34,7 +34,8 @@ projects=($@)
 if [ ${#projects[@]} -eq 0 ]
 then
     projects=(
-        "en_us/course_authors"
+       "en_us/contribute"
+       "en_us/course_authors"
```
Comment (jmbowman) [inline run_tests.sh:38]: The indentation here looks a little off.

**Review by jmbowman — COMMENTED** [review]
One minor nit, otherwise looks good after rebasing to fix the conflict.

**Comment by stvstnfrd** [comment]
Hi @grantgoodmanedX  Are you still interested in this or should we close it out for now?

**Comment by Carlos-Muniz** [comment]
@grantgoodmanedX Thank you very much for your contribution. However due to inactivity, I am closing this PR. Feel free to reopen and complete this work at anytime. Reach out in Slack if you need any help or guidance.

**Comment by openedx-webhooks** [comment]
Although this pull request is already closed, I've created [OSPR-6590](https://openedx.atlassian.net/browse/OSPR-6590) so that we can track it in Jira.

There is nothing you have to do. No action is needed from your side.  Thanks.

**Pull request closed without merging**